docs/proposal-config-changes.md

```diff
@@ -392,25 +396,49 @@ Matches global config change to `[task events]mail retry delays`.
 See
 [discussion](https://github.com/cylc/cylc-flow/issues/3404#issuecomment-540190164).
 
-### `[scheduler][events]abort if any task fails` -> `[scheduler][events]abort if any unexpected task fails`
+### `[scheduler][events]abort if any task fails`  -> obsolete
 
-See change above (different name proposed).
+Make command line option instead (discussed at CylcCon).
 
-### `[cylc][environment]` -> `[scheduler][environment]`
+### `[cylc][environment]` -> obsolete
 
-Makes sense to move this.
-
-Rename to `handler environment`?
-(think this was proposed but can't find discussion)
+Discussed at CylcCon.
 
 ### `[cylc]health check interval` -> `[scheduler]health check interval`
 
 Matches global config change.
 
+### `[cylc]` time settings -> `[scheduler]`
+
+This covers `UTC mode`, `cycle point format`,
+`cycle point num expanded year digits` and `cycle point time zone` (discussed at
+CylcCon).
+
+### `[cylc][simulation]disable suite event handlers` -> ???
+
+Agreed at CylcCon to move this and make it positive.
+Notes suggest `[scheduling]use suite event handlers`.
+However current setting only applies to simulation and dummy mode (and defaults to true) -
+we don't need this setting for real workflows do we (and we want these handles disabled by default in simulation and dummy mode)?.
+How about `[scheduler][events]enable simulation handlers`?
+
+### `[runtime][]extra log files` -> obsolete
+
+Instead provide access to all files found in the job log dir in the new UI. See
+[discussion](https://github.com/cylc/cylc-flow/pull/3348#discussion_r334219315).
+
+### Empty string defaults
+
+Where default values are currently defined as empty strings, these will be
+replaced with `None`. See
```

Comment:
Actually I'm not sure about this. Some config settings don't have a default defined whereas others have an empty string ('') - what is the difference?

Reply:
At the moment if you don't specify a default you get `None` though there is some logic to provide `[]` if that is more appropriate.

I have tweaked this a little in https://github.com/cylc/cylc-flow/pull/3559, now the configuration can tell the difference between a setting with no default and one with a default, though unset defaults still get expanded to `None` or `[]`.

Reply:
I'm not in any hurry to replace `''` with `None` though it would be nicer if all unset values defaulted to `None`.

Reply:
Happy to be corrected but from my understanding None is nonetype - not the same as an empty string. If it is going to be used for comaprison, 'none' means not yet set (or no value passed to the optional parameter), whereas "" might be a valid option.
From reading around None seems to be used by default more frequently. Although if you want to use string methods on it (regarless of whether it is there or not), then "" is better as if you use a sting method on None it will throw exceptions.
I think I remember reading somewhere that None can be good as if it isn't there isn't any memory assigned to it to it, whereas "" would - can't confirm this though - again happy to be corrected!

Reply:
That's right it's just a matter of how we use the settings.

Using empty strings as default values makes things a little easier in some ways:

```python
cmd = f'{ cfg["command-prefix"] } mail { cfg["mail opts"] } -s foo'
```

Whereas when using `None` defaults:

```python
cmd = f'{ cfg["command-prefix"] or "" } mail { cfg["mail opts"] or "" } -s foo'
```

Reply:
#3508 resulted from getting this sort of thing wrong (although that particular example will be irrelevant in the "Platforms" era.   Sometimes you need to know if an item has been explicitly been set to "nothing" (the exact value of which depends on its type) or not set at all.

Reply:
Sounds like this convo is resolved (unresolve if not!)